### PR TITLE
Add libblosc-dev

### PR DESCRIPTION
**Pull request by Timple**
Please add the following dependency to the rosdep database.

## Package name:

`libblosc-dev`

## Package Upstream Source:

https://github.com/Blosc/c-blosc

## Purpose of using this:

For: https://github.com/SteveMacenski/spatio_temporal_voxel_layer/pull/288

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/buster/libblosc-dev
- Ubuntu: https://packages.ubuntu.com/
  - https://packages.ubuntu.com/search?keywords=libblosc-dev
- Fedora: https://packages.fedoraproject.org/
  - https://packages.fedoraproject.org/pkgs/blosc/blosc-devel/index.html
- Arch: https://www.archlinux.org/packages/
  - IF AVAILABLE
- Gentoo: https://packages.gentoo.org/
  - IF AVAILABLE
- macOS: https://formulae.brew.sh/
  - IF AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/packages
  - IF AVAILABLE
- NixOS/nixpkgs: https://search.nixos.org/packages
  - OPTIONAL
- openSUSE: https://software.opensuse.org/package/
  - IF AVAILABLE
- rhel: https://rhel.pkgs.org/
  - IF AVAILABLE

# Please Add This Package to be indexed in the rosdistro.

Iron, humble, rolling, jazzy

# The source is here:

https://github.com/Blosc/c-blosc

# Checks
 - [ ] All packages have a declared license in the package.xml
 - [ ] This repository has a LICENSE file
 - [ ] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: rosdep/base.yaml
Comment (clalancette) [inline rosdep/base.yaml:2601]:
```suggestion
libblosc-dev:
  alpine: [blosc-dev]
  debian: [libblosc-dev]
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: rosdep/base.yaml
Comment (Timple) [reply rosdep/base.yaml]: Thanks!

File: rosdep/base.yaml
Comment (Timple) [reply rosdep/base.yaml:2603]: Thanks for the help!

**Review by clalancette — APPROVED** [review]

**Comment by Timple** [comment]
So who's now in charge of hitting the merge button? :slightly_smiling_face:

**Review by nuclearsandwich — APPROVED** [review]

**Comment by nuclearsandwich** [comment]
> So who's now in charge of hitting the merge button? 🙂

rosdep database changes require two reviewers: https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md so the merge was blocked by the second review.

**Comment by Timple** [comment]
> > So who's now in charge of hitting the merge button? 🙂
> 
> rosdep database changes require two reviewers: https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md so the merge was blocked by the second review.

Didn't know, learning everyday!

**Pull request merged**
